bucket_56/libproxy/specification

```diff
@@ -0,0 +1,66 @@
+DEF[PORTVERSION]=	0.4.15
+# ----------------------------------------------------------------------------
+
+NAMEBASE=		libproxy
+VERSION=		${PORTVERSION}
+KEYWORDS=		net devel
+VARIANTS=		standard minimal
+SDESC[standard]=	Library for automatic proxy configuration mgmt
+SDESC[minimal]=		Libproxy without any additional bindings
+HOMEPAGE=		https://libproxy.github.io/libproxy
+CONTACT=		Michael_Reim[kraileth@elderlinux.org]
+
+DOWNLOAD_GROUPS=	main
+SITES[main]=		https://github.com/libproxy/libproxy/archive/
+DISTFILE[1]=		${PORTVERSION}.tar.gz:main
```

Comment:
Intentional to use an archive tarball over a generated one (e.g. generated:main) ?

Reply:
With the old version that I was original trying to port I somehow couldn't get this to work. I just tried again and now the generated distfile is all fine.